proof/crefine/ARM_HYP/Arch_C.thy

```diff
@@ -1664,30 +1664,36 @@ lemma performPageInvocationMapPTE_ccorres:
              apply (rule conjI, clarsimp)
               apply (clarsimp split: if_split_asm)
                apply (case_tac a; clarsimp simp: isLargePagePTE_def)
-               apply (clarsimp simp: cpte_relation_def pte_lift_small split del: split_of_bool)
+               apply (clarsimp simp: cpte_relation_def pte_lift_small
+                               split del: split_of_bool)
```

Comment:
why all these? there's no difference to original except reindenting

Reply:
if you're going this far, a `supply` might make more sense?

Reply:
This is a result of having added `split del: split_of_bool_asm` before and then removing it again (moving it to a bundle). Better would be to revert that as well, but I didn't find any good way of getting just the bits that changed because of add/remove cycle.